index.rst

```diff
@@ -1,57 +1,344 @@
+        - Troubleshooting of systems will utilize component EUIs and feedback presented from LOVE and/or the scriptQueue
+        - Mining of information and analyzing of sequencing from the EFD is not expected
+        - Ability to enter and observe Chronograf dashboards is expected
+        - Possess ability to execute and make small edits to notebooks
+
+            - Requires a minimal level of Python, and knowledge of few commands of git (e.g. git-checkout and git-pull)
+            - Comfortable in finding and executing commands via high-level classes
+
+        - Ability to update and maintain operations related documentation (written in rST, hosted on GitHub)
+        - Some knowledge of low-level CSC functionality
+
+            - Able to examine and change between configurations
+            - Troubleshoot at the level diagnostics (error codes), status, and manual (non-DDS) motion where required
+
+        - Interacts with data to perform offsetting, focus, but via tooling that provides the calculations
+        - All information needed to operate the facility is provided to them, they are not required to develop analysis and/or display tools
+
+    - **Commissioning Scientist/Engineer**
+
+        - Includes operator control-software skills plus the following:
+        - Extensive use of the notebook interface, including the writing of code and launching of scripts
+
+            - Comfortable in Python, competent with git and GitHub
+
+        - Ability to diagnose both system and component level behavioural issues
+        - Not required to identify the issue in the source code
+        - Able to create and load new config files
+        - Writes and executes custom external scripts from both notebooks, ScriptQueue and LOVE
+        - Not expected to write production level scripts (see tstn-010 for definition)
+        - Able to switch between software versions of deployed components
+        - Able to update scriptQueue container repositories
+        - Able to diagnose issues via the EFD/Chronograf
+        - Ability to generate and maintain documentation (written in rST)
+        - Works out of already defined environments (e.g. NTS or Summit)
+
+            - Comfortable changing between software packages in Nublado environment
+        - Not expected to be familiar with software builds or deployment
+        - Not expected to work with the standard development container/environment
+
+    - **Software Developer**
+
+        - Diagnoses behavioural issues at the code-level of CSCs or higher-level classes
+        - Writes and reviews production level scripts
+
+            - Expert in Python, git and other applicable languages
+
+        - Modifies and builds components, tags for release where appropriate
+        - Familiar with deployment strategies and restarting components (ArgoCD)
+        - Ability to probe into individually deployed containers (Rancher)
```

Comment:
Although the majority of our CSCs are deployed with containers on k8s we do have some special snowflakes out there. Maybe just state "components" in general?

> - Ability to probe into individually deployed components, in the kubernetes cluster and/or bare metal machines